### PR TITLE
Update Cloudflare Billing Policy - preauths

### PR DESCRIPTION
**Pull request by abstergail**
### Documentation checklist

- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [ ] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

**Review by travisgahn — COMMENTED** [review]
Looks good on my end

File: content/support/account-management-billing/Billing Cloudflare Plans/Cloudflare Billing Policy.md [hunk 1]
```diff
@@ -105,7 +105,9 @@ ___
 
 ## Account Payment Method Preauthorization
 
-For Services subject to usage-based billing, Cloudflare may initiate a preauthorization on your credit card at any point in a billing period to confirm the payment method on file can cover accrued fees. Failure to pass preauthorization may result in suspension or termination of your account and/or access to the Services until your payment method can be validated.
+For services subject to usage-based billing, Cloudflare may preauthorize your credit card at any point in a billing period to confirm the payment method on file can cover accrued fees. If your payment method succeeds, service will continue normally.
+
+If your payment method fails, we will remove access to any usage-based billing services that use preauthorization (for example, R2). To regain access, you must settle any outstanding balances and pass preauthorization with a new, valid payment method. For assistance, [contact support](/support/contacting-cloudflare-support).
```
Comment (abstergail) [inline content/support/account-management-billing/Billing Cloudflare Plans/Cloudflare Billing Policy.md]:
```suggestion
If your payment method fails, we will remove access to any usage-based billing services that use preauthorization. In the case of [R2](/r2/), you will not be able to access your R2 buckets and requests will return errors, but your data will remain secure and unaffected.

To regain access, you must settle any outstanding balances and pass preauthorization with a new, valid payment method. To update your primary payment method, navigate to the Manage Account -> Billing -> Payment section of your account. Upon confirmation of your updated payment details, we will promptly reactivate your subscription(s), which will restore access to your data and services.

For assistance, visit our [Support Portal](https://support.cloudflare.com/hc/en-us) and submit a Billing request (category: “Payment issue”) to our Support team. They will assist you in verifying your updated payment information.
```

File: content/support/account-management-billing/Billing Cloudflare Plans/Cloudflare Billing Policy.md [hunk 2]
```diff
@@ -105,7 +105,13 @@ ___
 
 ## Account Payment Method Preauthorization
 
-For Services subject to usage-based billing, Cloudflare may initiate a preauthorization on your credit card at any point in a billing period to confirm the payment method on file can cover accrued fees. Failure to pass preauthorization may result in suspension or termination of your account and/or access to the Services until your payment method can be validated.
+For services subject to usage-based billing, Cloudflare may preauthorize your credit card at any point in a billing period to confirm the payment method on file can cover accrued fees. If your payment method succeeds, service will continue normally.
```
Comment (abstergail) [inline content/support/account-management-billing/Billing Cloudflare Plans/Cloudflare Billing Policy.md]:
```suggestion
For services subject to usage-based billing, Cloudflare may pre authorize your credit card at any point in a billing period to confirm the payment method on file can cover accrued fees. This is a temporary hold and you will not be charged until the end of your billing period. If your payment method is validated, service will continue normally.
```

File: content/support/account-management-billing/Billing Cloudflare Plans/Cloudflare Billing Policy.md [hunk 3]
```diff
@@ -105,7 +105,13 @@ ___
...
+
+If your payment method fails, we will remove access to any usage-based billing services that use preauthorization. In the case of [R2](/r2/), you will not be able to access your R2 buckets and requests will return errors, but your data will remain secure and unaffected.
```
Comment (abstergail) [inline content/support/account-management-billing/Billing Cloudflare Plans/Cloudflare Billing Policy.md]:
```suggestion
If your payment method fails, we may suspend your access to the usage-based billing services for which we conducted the preauthorization. In the case of [R2](/r2/), you will not be able to access your R2 buckets and requests will return errors, but your data will remain secure. If you do not update your payment method within 30 days, the data related to any usage-based billing service(s) may be deleted.
```

File: content/support/account-management-billing/Billing Cloudflare Plans/Cloudflare Billing Policy.md [hunk 4]
```diff
@@ -105,7 +105,13 @@ ___
...
+
+To regain access, you must settle any outstanding balances and pass preauthorization with a new, valid payment method. To update your primary payment method, navigate to the Manage Account -> Billing -> Payment section of your account. Upon confirmation of your updated payment details, we will promptly reactivate your subscription(s), which will restore access to your data and services.
```
Comment (abstergail) [inline content/support/account-management-billing/Billing Cloudflare Plans/Cloudflare Billing Policy.md]:
```suggestion
To regain access, you must settle any outstanding balances and pass preauthorization with a valid payment method. To update your primary payment method, navigate to the Manage Account -> Billing -> Payment section of your account. Upon validation of your updated payment details, we will promptly reactivate your subscription(s), which will restore access to the relevant data and services.
```

File: content/support/account-management-billing/Billing Cloudflare Plans/Cloudflare Billing Policy.md [hunk 5]
```diff
@@ -105,7 +105,13 @@ ___
...
+For services subject to usage-based billing, Cloudflare may pre authorize your credit card at any point in a billing period to confirm the payment method on file can cover accrued fees. This is a temporary hold and you will not be charged until the end of your billing period. If your payment method is validated, service will continue normally.
+
+If your payment method fails, we may suspend your access to the usage-based billing services for which we conducted the preauthorization. In the case of [R2](/r2/), you will not be able to access your R2 buckets and requests will return errors, but your data will remain secure. If you do not update your payment method within 30 days, the data related to any usage-based billing service(s) may be deleted.
+
+To regain access, you must settle any outstanding balances and pass preauthorization with a valid payment method. To update your primary payment method, navigate to the Manage Account -> Billing -> Payment section of your account. Upon validation of your updated payment details, we will promptly reactivate your subscription(s), which will restore access to the relevant data and services.
```
Comment (ranbel) [inline content/support/account-management-billing/Billing Cloudflare Plans/Cloudflare Billing Policy.md]:
```suggestion
To regain access, you must settle any outstanding balances and pass preauthorization with a valid payment method. To update your primary payment method, navigate to the **Manage Account** > **Billing** > **Payment** section of your account. Upon validation of your updated payment details, we will promptly reactivate your subscription(s), which will restore access to the relevant data and services.
```

File: content/support/account-management-billing/Billing Cloudflare Plans/Cloudflare Billing Policy.md [hunk 6]
```diff
@@ -105,7 +105,13 @@ ___
 
 ## Account Payment Method Preauthorization
 
-For Services subject to usage-based billing, Cloudflare may initiate a preauthorization on your credit card at any point in a billing period to confirm the payment method on file can cover accrued fees. Failure to pass preauthorization may result in suspension or termination of your account and/or access to the Services until your payment method can be validated.
+For services subject to usage-based billing, Cloudflare may pre authorize your credit card at any point in a billing period to confirm the payment method on file can cover accrued fees. This is a temporary hold and you will not be charged until the end of your billing period. If your payment method is validated, service will continue normally.
```
Comment (ranbel) [inline content/support/account-management-billing/Billing Cloudflare Plans/Cloudflare Billing Policy.md]:
```suggestion
For services subject to usage-based billing, Cloudflare may preauthorize your credit card at any point in a billing period to confirm the payment method on file can cover accrued fees. This is a temporary hold and you will not be charged until the end of your billing period. If your payment method is validated, service will continue normally.
```

**Review by ranbel — APPROVED** [review]

**Pull request merged**